packages/server/src/csp.ts

```diff
@@ -75,19 +87,21 @@ const localDirectives = Object.entries(directives).reduce(
   (acc, [key, value]) => {
     return {
       ...acc,
-      [key]: Array.isArray(value) ? [...value, 'localhost:*'] : value,
+      [key]: Array.isArray(value) ? [...value, 'localhost:* ws:'] : value,
```

Comment:
ws: for å få vite til å fungere når man jobber direkte på dekoratøren. Sikkert en bedre måte å løse det på 😄